ee/fe/modules/500-basic-auth/templates/deployment.yaml

```diff
@@ -44,7 +44,7 @@ spec:
       containers:
       - name: nginx
         {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 8 }}
-        image: "{{ $.Values.global.modulesImages.registry }}:{{ $.Values.global.modulesImages.tags.basicAuth.nginx }}"
+        image: {{ include "helm_lib_module_image" (list $ "nginx") }}
```

Comment:
I'll be surprised if there won't be problems with quotations. Due to ':' symbol in image string.

Reply:
It works when the colon symbol is in the middle of a string, e.g., `a:b`.

Reply:
@nabokihms says it right